app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -295,6 +295,9 @@ textarea {
     position: relative;
     vertical-align: top;
 
+    &:hover {
+      color: #5e9546;
```

Comment:
Color literals like `#5e9546` should only be used in variable declarations; they should be referred to via variable everywhere else.